memory/memory/memory.py

```diff
@@ -0,0 +1,100 @@
+import aioredis
+import asyncio
+import base64
+import concurrent
+import json
+import logging
+import os
+import uvloop
+from aiohttp import web
+import kubernetes_asyncio as kube
+
+from hailtop.config import get_deploy_config
+from hailtop.google_storage import GCS
+from hailtop.hail_logging import AccessLogger
+from hailtop.tls import get_in_cluster_server_ssl_context
+from hailtop.utils import AsyncWorkerPool, retry_transient_errors
+from gear import setup_aiohttp_session, rest_authenticated_users_only
+
+uvloop.install()
+
+BATCH_PODS_NAMESPACE = os.environ['HAIL_BATCH_PODS_NAMESPACE']
+log = logging.getLogger('batch')
+routes = web.RouteTableDef()
+
+socket = '/tmp/redis.sock'
+
+@routes.get('/healthcheck')
+async def healthcheck(request):  # pylint: disable=unused-argument
+    return web.Response()
+
+@routes.post('/getfile')
+@rest_authenticated_users_only
+async def getfile(request, userdata):
+    body = await request.json()
+    billing_project = body['billing_project']
+    filename = body['filename']
+    userinfo = await get_or_add_user(request.app, userdata, billing_project)
+    body = await get_file_or_none(request.app, userdata['username'], userinfo, filename)
+    if body is None:
+        raise web.HTTPNotFound()
+    print(f'{filename} got result of length { len(body) }')
```

Comment:
No space in around {}s in Python f-strings.

Remove or convert to logging message.